lib/harpy_widgets/theme/harpy_theme.dart

```diff
@@ -23,6 +24,23 @@ const ShapeBorder kDefaultShapeBorder = RoundedRectangleBorder(
 const BorderRadius kDefaultBorderRadius = BorderRadius.all(kDefaultRadius);
 const Radius kDefaultRadius = Radius.circular(16);
 
+/// Fonts
+const List<String> kAssetFonts = [
+  kDefaultBodyFontFamily,
+  kDefaultDisplayFontFamily,
+];
```

Comment:
```suggestion
const List<String> _assetFonts = [
  kDefaultBodyFontFamily,
  kDefaultDisplayFontFamily,
];
```

& move to the bottom of the file